style.css

```diff
@@ -0,0 +1,247 @@
+
+body {
+  box-sizing: border-box;
+  width: 100vw;
+  height: 100%;
+}
+
+.navbar {
+  padding-top: 3%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: static;
+  padding-left: 6%;
+  padding-right: 6%;
+}
+
+.logo {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  text-decoration: none;
+}
+
+.hamburger-btn {
+  background: none;
+  border: none;
+  padding: 0;
+  padding-right: 7%;
+  cursor: pointer;
+}
+
+button > .toggler-icon {
+  display: block;
+  background-color: #6070ff;
+  width: 16px;
+  height: 2px;
+  margin-bottom: 4px;
+  color: white;
+}
+
+.about-section {
+  padding-top: 20%;
+  width: 97%;
+  height: 100%;
+  background-image: url("./assets/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please stick to the Figma design but still your design is not same as Figma design if you check your design the background image is not matched with Figma design so kindly fix it :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-01 23-39-39](https://github.com/CobbyElsonfx/Portfolio-Website/assets/45226848/a3e7bd16-3d0b-4a97-9405-b7e69fa5caf6)

**Figma design**
![Screenshot from 2023-06-01 23-39-15](https://github.com/CobbyElsonfx/Portfolio-Website/assets/45226848/aaceb311-32a0-4d8d-aae4-85cfc8f83ef9)